tests/components/test_piezo.py

```diff
@@ -53,3 +55,25 @@ def test_piezo_interface_implementation():
 def test_piezo_instantiation():
     """Test that we can instantiate an piezo."""
     Piezo(0, MockPiezoBoard(), MockPiezoDriver())
+
+
+def test_piezo_interface_class_method():
+    """Tests piezo's interface_class method."""
+    piezo = Piezo(0, MockPiezoBoard(), MockPiezoDriver())
+    assert piezo.interface_class() is PiezoInterface
+
+
+def test_piezo_buzz_method():
+    """Tests piezo's buzz method."""
```

Comment:
```suggestion
    """Tests piezo's buzz method's input validation."""
```

This test does not check that the backend's buzz method is actually called with the correct parameters, so the name and docstring of the test is a bit misleading. I would be in favour of merging this test into `test_piezo_buzz_invalid_value`